<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DDatabaseInterfaceClass</class>
 <widget class="QWidget" name="DDatabaseInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>database interface</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#mWidgetPreview
{
background-color: rgb(223, 223, 223);
}

#mWidgetSearch
{
background-color: rgb(182, 182, 182);
}

QTableView{  
    color: rgb(0, 0, 0);  
    border: 1px solid #C07010;          /*边框颜色*/  
    gridline-color:#C07010;             /*grid线颜色*/
	border-color: rgb(143, 143, 143);  
	background-color: rgb(148, 148, 148);
    alternate-background-color: rgb(200, 200, 200); /*行交替颜色*/  
    selection-background-color: rgb(130, 190, 100); /*选中行背景颜色*/  
}  
  
QTableView::item:!alternate:!selected{  
    background-color: rgb(220, 220, 220);    /*交替行的另一颜色*/  
}

QHeaderView::section{  
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(80, 80, 80, 255), stop:1 rgba(30, 30, 30, 255));  
    color: rgb(240, 240, 240);  
    padding-left: 4px;  
    border: 1px solid #C07010;  
    min-height: 30px;  
}  

QPushButton
{
	background-color: rgb(85, 170, 127);
	font-weight:bold;
	border-width:0px;
}

QLineEdit
{
	border-radius:5px;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="mWidgetSearch" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="mLIneEditSql">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>  sql: Select * from assets</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mBtnSearch">
            <property name="text">
             <string>Find</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mBtnUpdateAll">
            <property name="text">
             <string>UpdateAll</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mBtnStartEdit">
            <property name="text">
             <string>Edit</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="mLineEditKeywords">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>  GEE assets keywords</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mBtnSearchOnline">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mBtnUpdateToDatabase">
            <property name="toolTip">
             <string>update result ofonline search to db </string>
            </property>
            <property name="statusTip">
             <string>update result ofonline search to db </string>
            </property>
            <property name="whatsThis">
             <string>update result ofonline search to db </string>
            </property>
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mBtnGo2Select">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Go</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="mComboKeyw">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="mWidgetView" native="true">
         <property name="minimumSize">
          <size>
           <width>700</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QTableView" name="mTableView">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mWidgetPreview" native="true"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DDatabaseInterface.qrc"/>
 </resources>
 <connections/>
</ui>
